### DISCUSSION TITLE
[New Feature] Skipping briefing animations with user input

### DISCUSSION THREAD

**Discussion by Rampastring** [Vinifera-Developers/Vinifera#1525] [Feature Requests]
### Description:
Briefing lines in the Tiberian Sun "Restate Briefing" screen appear painfully slow at times, especially when you're testing a mission and you've already seen the briefing a dozen times. It would be nice if you, as the player, would be able to entirely skip the animation and make all of the lines appear instantly with user input (like with a mouse click).

**Comment by CCHyper** [comment]
I think detecting the mouse click (any button) would be an ideal solution for this, and I don't see any harm in it being enabled by default as a QoL improvement.
